tests/unit/test_install.py

```diff
@@ -1173,6 +1173,8 @@ def test_runs_upgrades_on_more_recent_version(ws, any_prompt):
     sql_backend = MockBackend()
     wheels = create_autospec(WheelsV2)
 
+    StatementExecutionBackend.execute = MockBackend().execute
```

Comment:
```suggestion
```
This is incorrect, don't do it

Reply:
This is mocked because sql_backend was not getting passed to upgrades.I have changed the upgrade to execute from workspace_client execution statement and removed this from here